packages/graph/hash_graph/Makefile.toml

```diff
@@ -10,6 +10,11 @@ CARGO_MAKE_CARGO_PROFILE = "production"
 
 [tasks.test]
 run_task = [
-    { name = ["yarn", "deployment-up", "recreate-db", "migrate-up", "test-task", "deployment-down"], condition = { env_true = ["CARGO_MAKE_CI" ] } },
+    { name = ["yarn", "deployment-up", "recreate-db", "migrate-up", "test-task", "test-task-integration", "deployment-down"], condition = { env_true = ["CARGO_MAKE_CI" ] } },
     { name = ["test-task"] }
 ]
+
+[tasks.test-task-integration]
```

Comment:
Can you please update the README to say how to run integration tests

Reply:
Added it to the readme. Also, I made it callable from `cargo make`. 2afb0f0742841b81d869d9f092fec019458639ab